Source/RMG/UserInterface/MainWindow.cpp

```diff
@@ -1106,6 +1108,7 @@ void MainWindow::on_Action_Help_HomePage(void)
 
 void MainWindow::on_Action_Help_About(void)
 {
+    this->aboutDialog->exec();
```

Comment:
Please don't put `aboutDialog` in the class members, just do
```cpp
Dialog::AboutDialog dialog(this);
dialog.exec();
```

And please remove the include from `MainWindow.hpp` and put it in `MainWindow.cpp`